letsencrypt/plugins/manual.py

```diff
@@ -130,7 +130,7 @@ def _perform_single(self, achall):
         if self.conf("test-mode"):
             logger.debug("Test mode. Executing the manual command: %s", command)
             # sh shipped with OS X does't support echo -n
-            if sys.platform == "darwin":
+            if sys.platform == "darwin":   # pragma: no cover
                 executable = "/bin/bash"
             else:
                 executable = None
```

Comment:
could we use `executable = "/bin/bash" if sys.platform == "darwin" else None` instead of `pragma: no cover`? Expression fits on one line, and we're sure darwin branch doesn't have a terrible typo.